manifests/m/Microsoft/AzureCLI/2.70.0/Microsoft.AzureCLI.installer.yaml

```diff
@@ -0,0 +1,45 @@
+# Created with komac v2.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Microsoft.AzureCLI
+PackageVersion: 2.70.0
+InstallerLocale: en-US
+ReleaseDate: 2025-03-04
+Installers:
+- Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: python.exe
+    PortableCommandAlias: az
+  InstallerUrl: https://github.com/Azure/azure-cli/releases/download/azure-cli-2.70.0/azure-cli-2.70.0-x64.zip
+  InstallerSha256: C0F9704EDA44DB393FB1652525E2902363EED22A63EDFE7F69C51EDD99955E00

```

Comment:
```suggestion
```

This looks wrong as it renames `python.exe` executable to `az`. Not sure how this made it's way in previous manifests as well